### ISSUE TITLE
No names/hover alt text read via screen reader on links within "What lessons..." section

### ISSUE THREAD

**Issue by logan-reynolds** [deraadt/moodle-block_completion_progress#53]
The links within the "What lessons have I viewed for this course?" section lack names which makes navigation via screen reader very difficult. In addition to this, the information that appears when a user hovers over a lesson is not read to screen reader users. They would have no way of easily accessing their progress and completion data.

Each of the links within this section should receive an aria-label that corresponds with the section's name. In addition to this, the other information regarding progress and whether or not a lesson has been completed should be included in an aria-describedby attribute.
